compiler/rustc_target/src/spec/x86_64_unknown_none_linuxkernel.rs

```diff
@@ -14,17 +14,20 @@ pub fn target() -> Target {
     base.pre_link_args.get_mut(&LinkerFlavor::Gcc).unwrap().push("-m64".to_string());
 
     Target {
-        // FIXME: Some dispute, the linux-on-clang folks think this should use "Linux"
-        llvm_target: "x86_64-elf".to_string(),
+        // FIXME: Some dispute, the linux-on-clang folks think this should use
+        // "Linux" We disagree because running *on* linux is nothing like
+        // running *as" linux, and historically the "os" component as has always
+        // been used to mean the "as" part.
```

Comment:
shouldn't this be 'the "on" part'?

Reply:
Yes! 

...Nothing like writing the opposite of what one meant to...

Thanks